tests/sanity/ignore-2.9.txt

```diff
@@ -699,6 +699,7 @@ plugins/modules/identity/keycloak/keycloak_clienttemplate.py validate-modules:do
 plugins/modules/identity/keycloak/keycloak_clienttemplate.py validate-modules:parameter-type-not-in-doc
 plugins/modules/identity/opendj/opendj_backendprop.py validate-modules:doc-missing-type
 plugins/modules/identity/opendj/opendj_backendprop.py validate-modules:parameter-type-not-in-doc
+plugins/modules/monitoring/airbrake_deployment.py pylint:collection-invalid-deprecated-version
```

Comment:
No need to add an entry to `ignore-2.9.txt`, since this is a 2.10-only test that fails.